README.md

```diff
@@ -1,2 +1,131 @@
 # Portfolio_Microverse
-A responsive website project that describe my achievements.
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
[OPTIONAL] It is required that you remove the Microverse logo from the readme template, but it could be that the previous reviewer didn't take note of it😉 Kindly take it away or put your logo if you have one🤗